gcc/rust/rust-diagnostics.h

```diff
@@ -124,4 +124,13 @@ struct Error
 };
 } // namespace Rust
 
+// rust_debug uses normal printf formatting, not GCC diagnostic formatting.
+void
+rust_debug (const Location location, const char *fmt, ...) ATTRIBUTE_PRINTF_2;
+
+template <typename ... Args> void
+rust_debug (const char *fmt, Args... args) {
+    rust_debug (Location (), fmt, args...);
+};
```

Comment:
`[...]/gcc/rust/rust-diagnostics.h:134:2: warning: extra ';' [-Wpedantic]`.

Reply:
Applied in a6054b0